include/dpp/permissions.h

```diff
@@ -80,38 +82,41 @@ enum permissions : uint64_t {
 using role_permissions = permissions;
 
 /**
- * @brief Represents a permission bitmask (refer to enum dpp::permissions) which are hold in an uint64_t
+ * @brief Represents a permission bitmask (refer to enum dpp::permissions) which are held in an uint64_t
  */
-class DPP_EXPORT permission {
-protected:
```

Comment:
The original inetntion of having this as protected is to prevent users from accessing the .value field, and only accessing it through its functions or implicit type conversion.